app/styles/components/_panel.scss

```diff
@@ -86,6 +86,10 @@ $panel-content-max-width: 500px;
   background-color: $Theme-color-white;
 }
 
+.Panel--solidLightBlue {
```

Comment:
I know that we already have a couple of modifiers named like this in here, but I think that this is wrong. The name should be tied to the `function` of the component, not to the `style`. If we give names based on style, as soon as it changes (let's say that this modifier should now put the background in a shade of `red`) the name stops to make sense.